tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -52,3 +56,20 @@ def test_check_user_response_organization(convert2rhel):
         # the Ctrl+d is used to terminate the process instead
         c2r.sendcontrol("d")
     assert c2r.exitstatus != 0
+
+
+def test_auto_attach_pool_submgr(convert2rhel):
+    single_pool_id = env.str("RHSM_SINGLE_SUB_POOL")
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {} --username {} --password {} --debug".format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_SINGLE_SUB_USERNAME"),
+            env.str("RHSM_SINGLE_SUB_PASSWORD"),
+        )
+    ) as c2r:
+        c2r.expect(
+            f"{single_pool_id} is the only subscription available, it will automatically be selected for the conversion."
+        )
+        c2r.sendcontrol("c")
```

Comment:
Maybe I would use the `d` character so it's consistent in the whole test file.

Reply:
done!